portingdb/htmlreport.py

```diff
@@ -729,6 +747,49 @@ def mispackaged():
         mispackaged=mispackaged,
     )
 
+
+def namingpolicy():
+    """Naming policy tracking.
+    """
+    db = current_app.config['DB']()
+    query = db.query(tables.Package)
+    total_packages = query.count()
+    query = query.join(tables.CollectionPackage)
+    query = query.filter(tables.CollectionPackage.collection_ident == 'fedora')
+    query = query.filter(tables.CollectionPackage.is_misnamed.is_(True))
+    total_misnamed = query.count()
+    total_correct = total_packages - total_misnamed
+
+    # Progress bar info.
+    progress = get_naming_progress_info(total_correct, total_misnamed)
+
+    # Statuses with number of packages
+    statuses = OrderedDict(
+        db.query(tables.Status, func.count(tables.Package.name))
+        .outerjoin(tables.Status.packages)
+        .join(tables.CollectionPackage)
+        .filter(tables.CollectionPackage.collection_ident == 'fedora')
+        .filter(tables.CollectionPackage.is_misnamed.is_(True))
+        .group_by(tables.Status.ident)
+        .order_by(tables.Status.order))
```

Comment:
Thanks for explanations. I already made it work without the column. This query looks complicated with all the joins, but it does the job, so I'll keep it. 
The rest of the comments are addressed.